mobile.js

```diff
@@ -0,0 +1,25 @@
+const popUp = document.querySelector('.hidden-menu');
+const ham = document.querySelector('.h-menu');
+const closeBtn = document.querySelector('.close-btn');
+const body = document.querySelector('.body');
+const choice = document.querySelector('.hidden-cont');
+
+ham.addEventListener('click', () => {
+  popUp.style.display = 'block';
+  ham.style.display = 'none';
+});
+
+closeBtn.addEventListener('click', () => {
+  popUp.style.display = 'none';
+  ham.style.display = 'block';
+});
+
+choice.addEventListener('click', () => {
+  popUp.style.display = 'none';
+  ham.style.display = 'block';
+});
+
+body.addEventListener('click', () => {
+  popUp.style.display = 'none';
```

Comment:
- [ ] Great work you did here on the menu popup 👏 . However, there are two sight chnages needed to get this to fit the Figma design.

- Firstly, your close button needs a little adjustment with the margin to the top
- Secondly, link options are not equally aligned. 
- [ ] I've made an image comparison m for you below, kindly view it and make the requested changes. you'll agree with me that the present design though good, will not give a visually appealing view to a user which leads to bad user experience:

## Your view:
![image](https://user-images.githubusercontent.com/50990925/223343728-b84fe03d-2143-4934-b541-0cb86dfc4231.png)

## Figma
![image](https://user-images.githubusercontent.com/50990925/223342146-3e7affa3-3142-405c-89de-02a2570507ec.png)